lib/gooddata/lcm/actions/synchronize_date_dimension.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+# (C) 2019-2020 GoodData Corporation
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeDateDimension < BaseAction
```

Comment:
Please rename to MigrateGdcDateDimension, file name migrate_gdc_date_dimension.rb.
Document: Migrate date dimension urn:gooddata:date or urn:custom:date to urn:custom_v2:date

Reply:
Yes, I renamed, and fixed all relation issues